harbor_exporter.go

```diff
@@ -408,8 +408,8 @@ func main() {
 	kingpin.Flag("harbor.insecure", "Disable TLS host verification.").Default("false").BoolVar(&exporter.insecure)
 	kingpin.Flag("harbor.pagesize", "Page size on requests to the harbor API.").Envar("HARBOR_PAGESIZE").Default("100").IntVar(&exporter.pageSize)
 	skip := kingpin.Flag("skip.metrics", "Skip these metrics groups").Enums(MetricsGroup_Values()...)
-	kingpin.Flag("cache.enabled", "Enable metrics caching.").Default("false").BoolVar(&exporter.cacheEnabled)
-	kingpin.Flag("cache.duration", "Time duration collected values are cached for.").Default("20s").DurationVar(&exporter.cacheDuration)
+	kingpin.Flag("cache.enabled", "Enable metrics caching.").Envar("HARBOR_CACHE").Default("false").BoolVar(&exporter.cacheEnabled)
```

Comment:
@thechristschn  can we rename the variable to `HARBOR_CACHE_ENABLED`?

Reply:
I already thought about it, but kept it short. I changed it now.